packages/fluentui/docs/src/prototypes/table/index.tsx

```diff
@@ -169,5 +184,8 @@ export default () => (
     <ComponentPrototype title="Table example 3" description="Table with popover and context menu ">
       <InteractiveTable />
     </ComponentPrototype>
+    <ComponentPrototype title="Table example 4" description="Responsive table hiding columns">
```

Comment:
is the title displayed somewhere? If yes, let's rename it to somethinglike Responsive Table

Reply:
Done